package.json

```diff
@@ -38,7 +39,7 @@
     "cli-format": "yarn workspace @lightdash/cli format",
     "lint": "run-s common-lint backend-lint frontend-lint e2e-lint cli-lint",
     "format": "run-s common-format backend-format frontend-format e2e-format cli-format",
-    "test": "run-s backend-test",
+    "test": "run-s common-test backend-test ",
```

Comment:
We should update our github action `.github/workflows/test-and-build.yml`  to use `yarn test` now that we have multiple packages with tests.

Reply:
You are right, I thought github actions were using `yarn test` already 👍